docs/frameworks/symfony.md

```diff
@@ -202,35 +157,48 @@ framework:
 If using Webpack Encore, you also need to add the following config at the end of `webpack.config.js`
 
 ```js
+// webpack.config.js
+
 if (Encore.isProduction()) {
     // Note the '/build' at the end of the URL
     Encore.setPublicPath('https://<bucket-name>.s3.amazonaws.com/build');
     Encore.setManifestKeyPrefix('build/')
 }
 ```
 
-Finally, you can compile assets for production in the `public` directory, then synchronize that directory to a S3 bucket:
+Then, you can compile assets for production in the `public` directory, and synchronize that directory to a S3 bucket:
 
 ```bash
 php bin/console assets:install --env prod
 # if using Webpack Encore, additionally run
 yarn encore production
-aws s3 sync public/ s3://<bucket-name>/ --delete --exclude index.php
+aws s3 sync public/ s3://<bucket-name>/ \
+  --delete \
+  --exclude index.php \
+  --exclude public/build/manifest.json \
+  --exclude public/build/entrypoint.json
 ```
 
-### Assets in templates
+Finally, you need to update the `serverless.yml` file to exclude the `assets` and `public/build` directories from deployment:
 
-For the above configuration to work, assets must be referenced in templates via the `asset()` helper:
+```yaml
+# serverless.yml
 
-```html
-<script src="{{ asset('js/app.js') }}"></script>
+package:
+  pattern:
+    - '!assets/**'
+    - '!public/build/**'
+    - 'public/build/manifest.json'
```

Comment:
maybe add `!public/bundles` that may be generated by `bin/console assets:install`